### PR TITLE
XSPD1-1629 Update payout create form - Security tokens section UI upgraded

### PR DESCRIPTION
**Pull request by kapildave07**
## Description

XSPD1-1629 Update payout create form - Security tokens section UI upgradedull request.

## Changes

- UI upgraded

## Attached Links

1. Jira ticket: https://investax.atlassian.net/browse/IXSPD1-1629
2. Documents:

## Checklist

- [x] Manual tests
- [x] The pull request doesn't affect existing feature

## For reviewers

- Ensure new code doesn't break existing features
- Reviewer shouldn't merge if there are pending unresolved comments

### CODE REVIEW

File: src/components/Checkbox/index.tsx [hunk 1]
```diff
@@ -52,10 +52,10 @@ export const Checkbox: FC<Props> = ({
       disabled={disabled}
       className="checkbox"
     >
-      <Flex style={{ gap: 8 }}>
+      <Flex style={{ gap: 8, alignItems: 'center' }}>
```
Comment (thi-investax) [inline src/components/Checkbox/index.tsx:55]: @kapildave07 , does it affect other pages if we have new inline styles here?

File: src/components/FilePreview/index.tsx [hunk 2]
```diff
@@ -56,3 +53,40 @@ export const FilePreview: FC<Props> = ({
...
+            onClick={(event) => {
+              event.preventDefault();
+              handleDeleteClick();
+            }}
+          />
+        </ButtonText>
+      )}
+    </PayoutPreviewImageWrapper>
+  );
+};
```
Comment (thi-investax) [inline src/components/FilePreview/index.tsx:92]: what is this component. could you please take a screenshot @kapildave07 ?

File: src/pages/KYC/common.tsx [hunk 3]
```diff
@@ -375,7 +375,7 @@ export const Uploader: FC<UploaderProps> = ({
       {files && files.length > 0 && (
         <Flex flexWrap="wrap">
           {files.map((file: any, index) => (
-            <FilePreview
+            <FilePreviewPayout
```
Comment (thi-investax) [inline src/pages/KYC/common.tsx:378]: @kapildave07 why do we change somethings in the kyc here?

File: src/components/FilePreview/index.tsx (hunk 2 above)
Comment (kapildave07) [reply src/components/FilePreview/index.tsx:92]: Screenshot 2024-07-31 at 7 17 59 PM

here is the FilePreviewPayout component  @thi-investax

File: src/components/Checkbox/index.tsx (hunk 1 above)
Comment (kapildave07) [reply src/components/Checkbox/index.tsx:55]: @thi-investax no it will not affected any other page.

File: src/pages/KYC/common.tsx (hunk 3 above)
Comment (kapildave07) [reply src/pages/KYC/common.tsx:378]: kyc page have a component that name is Uploader we are using this for file upload. Previously we were using the same i added some conditional rendering here.

Comment (thi-investax) [reply src/pages/KYC/common.tsx]: if this is reused in multiple features, could we have a more generic name for it @kapildave07 ? KYC shouldn't know about Payout domain. it violates the separation of concern rule in the SOLID principle

Comment (kapildave07) [reply src/pages/KYC/common.tsx]: @thi-investax sure i refactored uploader component.

**Pull request merged**